src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/TemplatingPass.php

```diff
@@ -25,8 +25,8 @@ public function process(ContainerBuilder $container)
         }
 
         if ($container->hasAlias('templating')) {
-            $definition = $container->findDefinition('templating');
-            $definition->setAutowiringTypes(array(ComponentEngineInterface::class, FrameworkBundleEngineInterface::class));
+            $container->setAlias(ComponentEngineInterface::class, 'templating');
+            $container->setAlias(FrameworkBundleEngineInterface::class, 'templating');
```

Comment:
this should be private aliases. We don't want to keep these service names in the compiled container